examples/mqtt/dcc_comms/iotcc/dell5k_edge_system_iotcc_mqtt.py

```diff
@@ -0,0 +1,192 @@
+# ----------------------------------------------------------------------------#
+
+from linux_metrics import cpu_stat, disk_stat, net_stat, mem_stat
+from liota.dccs.iotcc import IotControlCenter
+from liota.lib.utilities.identity import Identity
+from liota.entities.metrics.metric import Metric
+from liota.entities.devices.simulated_device import SimulatedDevice
+from liota.entities.edge_systems.dell5k_edge_system import Dell5KEdgeSystem
+from liota.dcc_comms.mqtt_dcc_comms import MqttDccComms
+from liota.dccs.dcc import RegistrationFailure
+
+# getting values from conf file
+config = {}
+execfile('sampleProp.conf', config)
+
+
+# some standard metrics for Linux systems
+# agent classes for different IoT system
+# agent classes for different data center components
+# agent classes for different kinds of of devices, connected to the IoT System
+# we are showing here how to create a representation for a Device in IoTCC but
+# using the notion of RAM (because we have no connected devices yet)
+# agent classes for different kinds of layer 4/5 connections from agent to DCC
+# -------User defined functions for getting the next value for a metric --------
+# usage of these shown below in main
+# semantics are that on each call the function returns the next available value
+# from the device or system associated to the metric.
+
+def read_cpu_procs():
+    return cpu_stat.procs_running()
+
+
+def read_cpu_utilization(sample_duration_sec=1):
+    cpu_pcts = cpu_stat.cpu_percents(sample_duration_sec)
+    return round((100 - cpu_pcts['idle']), 2)
+
+
+def read_disk_usage_stats():
+    return round(disk_stat.disk_reads_writes('sda')[0], 2)
+
+
+def read_network_bytes_received():
+    return round(net_stat.rx_tx_bytes('ens33')[0], 2)
+
+
+def read_mem_free():
+    total_mem = round(mem_stat.mem_stats()[1], 4)
+    free_mem = round(mem_stat.mem_stats()[3], 4)
+    mem_free_percent = ((total_mem - free_mem) / total_mem) * 100
+    return round(mem_free_percent, 2)
+
+
+# ---------------------------------------------------------------------------
+# In this example, we demonstrate how System health and some simulated data
+# can be directed to data center components IoTCC using MQTT from Liota.
+# The program illustrates the ease of use Liota brings to IoT application developers.
+
+if __name__ == '__main__':
+
+    # create a data center object, IoTCC in this case, using MQTT as a transport layer
+    # this object encapsulates the formats and protocols necessary for the agent to interact with the dcc
+    # UID/PASS login for now.
+
+    #  Creating EdgeSystem
+    edge_system = Dell5KEdgeSystem(config['EdgeSystemName'])
+    #  Encapsulates Identity
+    identity = Identity(root_ca_cert=None, username=config['broker_username'], password=config['broker_password'],
+                        cert_file=None, key_file=None)
+
+    iotcc = IotControlCenter(config['broker_username'], config['broker_password'],
```

Comment:
Agreed. It will require change in all the examples, can be taken up in next pull request. If you are open, you can take it next pull request.